pkg/capability/capability.go

```diff
@@ -12,6 +15,69 @@ type Capability struct {
 
 type CapabilityList []Capability
 
+// A list of known capabilities and notes on enabling them can be found on
+// https://docs.google.com/spreadsheets/d/1XMFkbG9DeXoBSOn5HPbzk-l2T_OjQdLjz452a6vaYCE
```

Comment:
This document is not publicly available, so I think we should remove the comment.